setup.sh

```diff
@@ -10,12 +18,26 @@ while [ $# -gt 0 ] ; do
 done
 
 if [ -z $GPU_ARCH ]; then
-    echo "Target CPU not specified"
-    echo "Syntax:"
-    echo "$> sh setup.sh -mcpu <gpu architecture>"
-    echo "For example:"
-    echo "$> sh setup.sh -mcpu sm_75"
-    return 1
+    echo -n "autodetect compute capability: "
+    GPU_ARCH=$(\
+        sh ./compute_capability.sh --expect-single --sm-prefix \
+        || true \
+    )
+    if [ -z "$GPU_ARCH" ]; then
+        echo "FAIL"
+
+        fatal $(cat <<EOF
+Target CPU not specified\n
```

Comment:
```suggestion
Target GPU not specified\n
```

Reply:
It was a typo in the original and matches the `-mcpu` option, so I would keep it, even though it is a bit weird, I agree

Reply:
Done in #93